packages/web-app-files/src/components/Search/List.vue

```diff
@@ -108,7 +108,13 @@
           :fields-displayed="['name', 'size', 'tags', 'mdate']"
           :resource-dom-selector="resourceDomSelector"
           @file-click="triggerDefaultAction"
-          @row-mounted="rowMounted"
+          @item-visible="
+            loadPreview({
+              space: getMatchingSpace($event),
+              resource: $event,
+              ignoreAlmostEmptyTxtFiles: true
```

Comment:
Huh, does this feature still exist? I thought we killed it.

Reply:
We ignore almost empty text files in the file lists apparently. Should I kill it? 😂

Reply:
Yeah I remember when we built the feature. But the added value is minimal while adding unnecessary complexity. So I thought we killed it already. If not I'd vote for removing the feature from the code base.